core-services/release-controller/_releases/README.md

```diff
@@ -0,0 +1,3 @@
+# Release Gating Jobs.
+
+This directory contains the configuration for [release gating and informing jobs](https://docs.ci.openshift.org/docs/architecture/release-gating/).  Any jobs marked gating must obtain approval from the [Technical Release Team](/OWNERS_ALIASES).
```

Comment:
to make the semantics of "marked gating" more clear, maybe:

> Any jobs without `optional: true` must obtain...

?

Reply:
Let me reword this...

Reply:
Check out the update.